services/src/main/java/org/exoplatform/task/service/impl/LabelServiceImpl.java

```diff
@@ -60,7 +65,15 @@ public LabelDto getLabel(long labelId) {
     @Override
     @ExoTransactional
     public LabelDto createLabel(LabelDto label) {
-        return labelStorage.createLabel(label);
+        LabelDto newLabel = labelStorage.createLabel(label);
+        try {
+            if(newLabel != null){
+                listenerService.broadcast(LABEL_CREATION, null,newLabel);
+            }
+        } catch (Exception e) {
+            LOG.error("Error while broadcasting label creation event", e);
```

Comment:
use LOG.warn, the operation of label creation isn't stopped, thus it's not a blocker exception.